res/res_pjsip_sdp_rtp.c

```diff
@@ -1967,6 +2048,28 @@ static int create_outgoing_sdp_stream(struct ast_sip_session *session, struct as
 		build_dtmf_sample_rates = 0;
 	}
 
+	/* if we are using text with rtp redundancy, we need to match the t140 payload inside the RED sdp */
+	for (index = 0; index < ast_format_cap_count(caps); ++index) {
+		struct ast_format *format = ast_format_cap_get_format(caps, index);
+		if (ast_format_get_type(format) == AST_MEDIA_TYPE_TEXT && strcasecmp(ast_format_get_name(format), "T140") == 0) {
+			if (session_media_transport != session_media) {
+				if ((rtp_code = ast_rtp_codecs_payload_code(ast_rtp_instance_get_codecs(session_media_transport->rtp), 1, format, 0)) == -1) {
+					ast_log(LOG_WARNING, "Unable to get rtp codec payload code for %s\n", ast_format_get_name(format));
```

Comment:
Log/debug messages here should use ast_sip_session_get_name too

Reply:
Changed, also adapted log message as we are now printing session name.